pkgs/development/ocaml-modules/gen_js_api/ojs.nix

```diff
@@ -0,0 +1,26 @@
+{ buildDunePackage
+, lib
+, ocaml
+, ppxlib
+, gen_js_api
+, fetchFromGitHub
+, js_of_ocaml-compiler
+}:
```

Comment:
It seems that a few arguments are not used: ocaml, ppxlib, fetchFromGitHub

Reply:
You are right, thanks!  I am accustomed to a compiler telling me so :)